automatic/chromium/tools/chocolateyInstall.ps1

```diff
@@ -1,11 +1,17 @@
-$packageName = '{{PackageName}}'
-$fileType = 'exe'
+  $packageArgs = @{
+  packageName   = '{{PackageName}}'
+  fileType      = 'exe'
+  url           = '{{DownloadUrl}}'
+  url64bit      = '{{DownloadUrlx64'
+  silentArgs    = $silentArgs
+  validExitCodes= @(0)
+  softwareName  = 'Chromium'
+  checksum      = '{{Checksum}}'
+  checksumType  = 'md5'
+  checksum64    = '{{Checksumx64}}'
+  checksumType64= 'md5'
+}
 $version = '{{PackageVersion}}'
-$systemIs64bit = Get-ProcessorBits
-$url = @{$true = "{{DownloadUrlx64}}"; $false = "{{DownloadUrl}}"}[$systemIs64bit -eq 64]
-$checksum = @{$true = "{{Checksumx64}}"; $false = "{{Checksum}}"}[$systemIs64bit -eq 64]

```

Comment:
This is short for having a literal in PowerShell, or a condensed version of a if then statement